prediction-applier-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-prediction-applier-extension/alfresco-global.properties

```diff
@@ -3,3 +3,14 @@
 ## System Administrators can override these values in environment specific configurations in
 ## alfresco/tomcat/shared/classes/alfresco-global.properties.
 ##
+hxi.auth.providers.hyland-experience.type=oauth2
+hxi.auth.providers.hyland-experience.grant-type=urn:hyland:params:oauth:grant-type:api-credentials
+hxi.auth.providers.hyland-experience.client-name=acs-insight-connector
+hxi.auth.providers.hyland-experience.client-id=
+hxi.auth.providers.hyland-experience.client-secret=
+hxi.auth.providers.hyland-experience.scope=hxp.integrations
+hxi.auth.providers.hyland-experience.token-uri=http://localhost:8001/token
+hxi.auth.providers.hyland-experience.environment-key=hxi-env-key
+
+hxi.client.baseUrl=http://localhost:8001
+hxi.client.askQuestionUrl=${hxi.client.baseUrl}/v1/questions
```

Comment:
I wouldn't expose `/v1/questions` - it gives us something extra to document and customers something extra to misconfigure.

Reply:
done